src/bidiMapper/domains/network/networkProcessor.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2023 Google LLC.
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Protocol from 'devtools-protocol';
+import {CdpClient} from '../../CdpConnection';
+import {IEventManager} from '../events/EventManager';
+import {LogType, LoggerFn} from '../../../utils/log';
+
+import {Network} from '../../../protocol/protocol.js';
+
+export class NetworkProcessor {
+  readonly #eventManager: IEventManager;
+  readonly #logger: LoggerFn | undefined;
+
+  static async init(
```

Comment:
For consistency with the rest of our codebase, prefer `create` instead of `init`

https://github.com/search?q=repo%3AGoogleChromeLabs%2Fchromium-bidi+static&type=code